Protocol/ErrorMessages.xml

```diff
@@ -9745,6 +9745,31 @@
 							</ErrorMessage>
 						</ErrorMessages>
 					</Check>
+					<Check id="76">
+						<Name namespace="Protocol.Params.Param.Interprete.Exceptions">CheckTypeTag</Name>
+						<ErrorMessages>
+							<ErrorMessage id="1">
+								<Name>ExceptionIncompatibleWithParamType</Name>
+								<GroupDescription />
+								<Description templateId="9000">
+									<InputParameters>
+										<InputParameter id="0" value="Interprete/Exceptions">item1Title</InputParameter>
+										<InputParameter id="1">item1Value</InputParameter>
```

Comment:
This argument name is very unclear what it need to be. I would suggest to use something like 'paramTypeValue'

Reply:
I set it to just paramType since paramTypeValue is not present anywhere else in the file.